PyVOTCA/molecule.py

```diff
@@ -50,6 +65,36 @@ def writeXYZfile(self, filename: Pathlike):
         with open(filename, "w") as xyzfile:
             xyzfile.write(mol)
 
+    def getTotalEnergy(self, kind, level='', dynamic = False):
+        """ Wraps call to individual total energy functions."""
+        if kind == 'dft_tot':
+            return self.getDFTEnergy()
+        elif kind == 'ks':
+            return self.getKSTotalEnergy(level)
+        elif kind == 'QPpert':
+            return self.getQPTotalEnergy(level)
+        elif kind == 'QPdiag':
+            return self.getQPTotalEnergy(level)
+        elif kind == 'BSE_singlet' and not dynamic:
+            return self.getBSEsingletTotalEnergy(level)
+        elif kind == 'BSE_singlet' and  dynamic:
+            return self.getBSEsingletDynamicTotalEnergy(level)
+        elif kind == 'BSE_triplet' and not dynamic:
+            return self.getBSEtripletTotalEnergy(level)
+        elif kind == 'BSE_triplet' and  dynamic:
+            return self.getBSEtripletDynamicTotalEnergy(level)
+        else:
+            raise Exception(
+                    f'Energy of kind {kind} is not available!')
+
+    def getGradient(self):
+        """Returns the stored nuclear gradient in Hartree/Bohr"""
+        if self.hasGradient:
+            return self.gradient
+        else:
+            raise Exception(
+                    f'Nuclear gradient not available!')
```

Comment:
the `f"` syntax is only necessary if you are going to format something inside the string